openy.install

```diff
@@ -841,3 +841,13 @@ function openy_update_8055() {
   $config_importer->setDirectory($openyLocFinderConfigDir);
   $config_importer->importConfigs(['views.view.locations.yml']);
 }
+
+/**
+ * Update Branch Hours field.
+ */
+function openy_update_8056() {
+  $config_importer = \Drupal::service('openy_upgrade_tool.importer');
+  $branchLocationConfig = drupal_get_path('module', 'openy_loc_branch') . '/config/install';
+  $config_importer->setDirectory($branchLocationConfig);
+  $config_importer->importConfigs(['field.storage.node.field_branch_hours.yml']);
```

Comment:
name shouldn't end with a file extension. This functionality not yet landed to confi module